scripts/docs-gen-snippets.mjs

```diff
@@ -0,0 +1,202 @@
+                    | Proxy requests | ✅ |
+                </Accordion>
+                <Accordion title="✅ Observability & data quality">
+                    | Tools | Status | 
+                    | - | - | 
+                    | HTTP request logging | ✅ |
+                    | End-to-type type safety | ✅ |
+                    | Data runtime validation | ✅ |
+                    | OpenTelemetry export | ✅ |
+                    | Slack alerts on errors | ✅ |
+                    | Integration status API | ✅ |
+                </Accordion>
+                <Accordion title="✅ Customization">
+                    | Tools | Status | 
+                    | - | - | 
+                    | Create or customize use-cases | ✅ |
+                    | Pre-configured pagination | ✅ |
+                    | Pre-configured rate-limit handling | ✅ |
+                    | Per-customer configurations | ✅ |
+                </Accordion>
+            </AccordionGroup>  
+        `
+        .split('\n')
+        .map((line) => line.trim())
+        .join('\n');
+}
+
+function useCasesSnippet({ useCases }) {
+    if (!useCases || useCases.length === 0) {
+        return emptyUseCases();
+    }
+
+    const groups = {};
+    for (const useCase of useCases) {
+        const groupName = useCase.group || 'Others';
+        useCase.group = groupName;
+
+        groups[groupName] = groups[groupName] || [];
+        groups[groupName].push(useCase);
+    }
+
+    const sortedGroups = Object.keys(groups)
+        .sort()
+        .reduce((acc, key) => {
+            acc[key] = groups[key];
+            return acc;
+        }, {});
+
+    // make sure Others is last
+    if (sortedGroups['Others']) {
+        const others = sortedGroups['Others'];
+        delete sortedGroups.Others;
+        sortedGroups.Others = others;
+    }
+
+    return `
+        ## Pre-built use cases
+        
+        <AccordionGroup>
+            
+            ${Object.values(sortedGroups)
+                .map(
+                    (group) => `
+                        <Accordion title="${group[0].group || 'Others'}">
+                        | Endpoint | Description | Readme |
+                        | - | - | - |
+                        ${group
+                            .map(
+                                (endpoint) =>
+                                    `| \`${endpoint.method} ${endpoint.path}\` | ${endpoint.description?.replaceAll('\n', '<br />')} | [github.com](https://github.com/NangoHQ/integration-templates/blob/main/integrations/${endpoint.script}.md) |`
+                            )
+                            .join('\n')}
+                        </Accordion>
+                `
+                )
+                .join('\n')}
+        </AccordionGroup>
+    `
+        .split('\n')
+        .map((line) => line.trim())
+        .join('\n');
+}
+
+function emptyUseCases() {
+    return `## Pre-built use-cases
```

Comment:
Why is it called use-cases and not just endpoints? That would resonate with the dashboard API ref since it's presenting the same data, cc @bastienbeurier

Reply:
Confirmed with @bastienbeurier (https://nangohq.slack.com/archives/C067NRBH1LL/p1734018441493369) that we'll ship now as-is and change later if needed.